Slim/Networking/Async/HTTP.pm

```diff
@@ -117,7 +126,21 @@ sub new_socket {
 			return Slim::Networking::Async::Socket::HTTP->new( %args );
 		}
 	}
-	else {
+	elsif ($self->socksAddr) {
+		my %args = @_;
+		main::DEBUGLOG && $log->debug("Using SOCKS proxy", $self->socksAddr, ":", $self->socksAddr);
+		# socks negotiation needs to be blocking or it terminates immediately on timeout
+		my $sock Slim::Networking::Async::Socket::HTTPsocks->new( @_, 
```

Comment:
Isn't this missing the `=` sign for the assignment?

Reply:
shame on me for bad process